joomla/joomla-cms
# [4.2] contacts - Indicate if category not published

### Summary of Changes
1. Adds an indicator if the category is unpublished or trashed.
2. Changes the published icon to a disabled icon and adds a tooltip to the icon
3. changes to use the publishedbutton and featuredbutton

### Testing Instructions
Create a category called test and create 4 featured contacts in that category.
Unpublish or Trash the category

Check the list of contacts

### Actual result BEFORE applying this Pull Request
No indication that the contact can not be displayed

### Expected result AFTER applying this Pull Request
Text as shown in the image below

![image](https://user-images.githubusercontent.com/1296369/165383669-d50a183f-94f0-4e6d-b2a4-cfa0c05faf40.png)
